eks/vpn.tf

```diff
@@ -1,7 +1,7 @@
 resource "aws_eip" "vpn_eip" {
   instance = aws_instance.vpn[0].id
   vpc      = true
-  count       = var.foxpass_api_key != "" ? 1 : 0
+  count    = var.foxpass_api_key != "" ? 1 : 0
```

Comment:
Why is this here? It should maae it regardless?

Reply:
It doesn't make a vpn available if the foxpass api key isnt set. Should I change it? I though the normal ipv4 of an ec2 instance isn't mapped to a vpc.

Reply:
Oh I see, Sorry was thinking LogDNA for some reason.